editoast/src/views/timetable/stdcm.rs

```diff
@@ -264,10 +272,20 @@ async fn stdcm(
         None => vec![],
     };
 
-    // 6. Build STDCM request
-    let stdcm_response = STDCMRequest {
+    // 6. Retrieve work schedules
+    let work_schedules = match stdcm_request.work_schedule_group_id {
+        Some(work_schedule_group_id) => {
+            let selection_setting: SelectionSettings<WorkSchedule> = SelectionSettings::new()
```

Comment:
```suggestion
            let selection_setting = SelectionSettings::new()
```

I think rust can infer the type here.

Reply:
Done.